plugins/http_plugin/include/eosio/http_plugin/beast_http_session.hpp

```diff
@@ -132,14 +130,16 @@ class beast_http_session : public detail::abstract_conn {
 
       // Request path must be absolute and not contain "..".
       if(req.target().empty() || req.target()[0] != '/' || req.target().find("..") != beast::string_view::npos) {
+         fc_ilog( plugin_state_->logger, "Return bad_reqest:  ${target}",  ("target", std::string(req.target())) );
```

Comment:
If this is kept it should be `fc_dlog`, but does it add anything to the `fc_dlog` in `send_response`?